common/src/main/java/io/netty/util/internal/logging/LocationAwareSlf4JLogger.java

```diff
@@ -54,28 +58,28 @@ public boolean isTraceEnabled() {
     @Override
     public void trace(String msg) {
         if (isTraceEnabled()) {
```

Comment:
Not part of this PR but I think methods like this one don't need the `if (isXxxEnabled())` right?